accounts/migrations/0001_initial.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-30 15:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import localflavor.us.models
+import timezone_field.fields
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0007_alter_validators_add_error_messages'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
+                ('created_on', models.DateTimeField(auto_now_add=True)),
+                ('updated_on', models.DateTimeField(auto_now=True)),
+                ('is_active', models.BooleanField(default=True, verbose_name='Active')),
+                ('is_staff', models.BooleanField(default=False, verbose_name='Staff')),
+                ('first_name', models.CharField(max_length=50, verbose_name='First Name')),
+                ('last_name', models.CharField(max_length=50, verbose_name='Last Name')),
+                ('email', models.EmailField(max_length=254, unique=True, verbose_name='Email')),
+                ('timezone', timezone_field.fields.TimeZoneField(default='America/New_York')),
+            ],
+            options={
+                'abstract': False,
+                'swappable': 'AUTH_USER_MODEL',
```

Comment:
@nkarmi can you add some documentation why did you use this and how it works, because as far as I understood this is an undocumented code from Django side